internal/grpctest/grpctest.go

```diff
@@ -58,11 +58,13 @@ func (Tester) Setup(t *testing.T) {
 	//  completely addressed, and this can be turned back on as soon as this issue is
 	//  fixed.
 	leakcheck.SetTrackingBufferPool(logger{t: t})
+	leakcheck.SetTimerTracker(logger{t: t})
 }
 
 // Teardown performs a leak check.
 func (Tester) Teardown(t *testing.T) {
 	leakcheck.CheckTrackingBufferPool()
+	leakcheck.CheckTimers(5 * time.Second)
```

Comment:
This is additive with the 10s for checking goroutines.  It might be nice to pass a context (or timer) to both of these instead, so they can share the 10s timeout?